nipype/interfaces/fsl/tests/test_auto_Eddy.py

```diff
@@ -38,9 +38,9 @@ def test_Eddy_inputs():
     mandatory=True,
     ),
     in_topup_fieldcoef=dict(argstr='--topup=%s',
-    requires=['in_topup_movpar'],
+    requires=[u'in_topup_movpar'],
```

Comment:
the primary reason we were using python 2 was because of unicode issues. however, if we include `unicode_literals` at the top of template, we can generate the same tests from either py2 or py3 by modifying the test generator a bit.

the main point of these auto-tests were to create an easy visual check for changes in input/outputspec api and to instantiate each object in the absence of consistent doctests. the former we can capture through a good PR review. the latter would require some work to go through all interfaces. so if we can achieve these two things somehow, it would be good.